### ISSUE TITLE
Could you show an example of upgrading when using the PostgreSQL Operator Installer.

### ISSUE THREAD

**Issue by pengbo0328** [CrunchyData/postgres-operator#1673]
The [documentation](https://access.crunchydata.com/documentation/postgres-operator/4.3.2/upgrade/automatedupgrade/) shows "the automated upgrading when using an Ansible installation of the PostgreSQL Operator" and " the automated upgrading when using a Bash installation of the PostgreSQL Operator".

However, if I installed postgres operator using the PostgreSQL Operator Installer ( `kubectl apply -f postgres-operator.yml` ), how can I upgrade the postgres operator and PG cluster.

I think `pgo upgrade` is to upgrade the PG cluster. How can I upgrade the PostgreSQL Operator.

Thanks.

**Comment by tjmoore4** [comment]
Yes, you are correct about the usage of the `pgo upgrade` command. 

Currently, the PostgreSQL Operator Installer utilizes the Ansible installer to accomplish the install/update/uninstall tasks. The relevant settings are mentioned under the "Configuring to Update and Uninstall" heading of the relevant [installer documentation](https://access.crunchydata.com/documentation/postgres-operator/4.3.2/installation/postgres-operator/.) Basically, you'll need to set the `DEPLOY_ACTION` environment variable in the postgres-operator.yml file to 'update', similar to the existing Ansible [update process](https://access.crunchydata.com/documentation/postgres-operator/4.3.2/installation/other/ansible/updating-operator/)

Sorry for any confusion, we will look into updating these instructions to better explain the upgrade process using the PostgreSQL Operator Installer.

**Comment by pengbo0328** [comment]
Thank you for your quick reply.
I will try the update process of setting `DEPLOY_ACTION` environment variable to `update` in the postgres-operator.yml.

**Comment by jkatz** [comment]
As there does not appear to be anything else, I will close out this issue, but please let us know if you have any more questions. Thanks!
